### ISSUE TITLE
getOnlyURL logic

### ISSUE THREAD

**Issue by micate** [krasimir/navigo#157]
If we pick up the `getOnlyURL` method, comment pushState logic, it would look like this:
```javascript
function getOnlyURL(url, useHash, hash) {
  var onlyURL = url.split(/\?(.*)?$/)[0], split;

  if (typeof hash === 'undefined') {
    // To preserve BC
    hash = '#';
  }

  // comment for hash case
  // if (isPushStateAvailable() && !useHash) {
  //   onlyURL = onlyURL.split(hash)[0];
  // } else {
    split = onlyURL.split(hash);
    onlyURL = split.length > 1 ? onlyURL.split(hash)[1] : split[0];
  // }

  return onlyURL;
}
```

Now if we pass in a param like `/employee?spm=abc#/employee/iparents`, the output is:
```
> getOnlyURL('/employee?spm=abc#/employee/iparents', true, '#')
/employee
```
But what I expected is `/employee/iparents`.

**Comment by micate** [comment]
#158

**Comment by krasimir** [comment]
Thank you for reporting the bug. Your solution was on the right track but it broke a couple of other unit tests. The problem is now solved and released as 5.3.2 version.
